docs/primitives.md

```
@@ -519,6 +519,9 @@ type TemplateContext = {
 	--- Object Storage plugin
 	ObjectStorage: objectstorageP.Plugin,
 
+	--- HTTP Client plugin
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)